common/window.js

```diff
@@ -97,3 +97,16 @@ export const isMobileBrowser = (userAgent) => {
 
   return !!mobile;
 };
+
+export const _checkDebounce = (func, wait) => {
```

Comment:
Why is this `_`? this syntax is preserved for usually a private method. Why not just call it `debounce` instead of `checkDebounce`?